src/vs/platform/terminal/common/terminal.ts

```diff
@@ -418,6 +418,8 @@ export interface ITerminalLaunchError {
 	code?: number;
 }
 
+export interface IProcessTraits { pid: number, cwd: string, requiresWindowsMode?: boolean }
```

Comment:
Let's call this IProcessReadyEvent for consistency with the other events, also properties on new line and `;` over `,`